internal/v1/handler_blueprints.go

```diff
@@ -62,6 +79,18 @@ func (bb *BlueprintBody) CryptPasswords() error {
 	return nil
 }
 
+func (bb *BlueprintBody) RedactPasswords() {
+	if bb.Customizations.Users != nil {
+		var redactedUsers []User
+		for _, u := range *bb.Customizations.Users {
+			u.RedactPassword()
+			redactedUsers = append(redactedUsers, u)
+		}
+		bb.Customizations.Users = &redactedUsers
+	}
+}
```

Comment:
Same comment as I was making in the other commit - you can do this without copying all the values.

Reply:
yep, fixed